components/form.html

```diff
@@ -5,31 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
     <link rel="stylesheet" href="./styles.css" />
+    
+    <script type="module" src="./form.js" defer></script>
   </head>
   <body>
     <header class="header">
         <h1 class="header__title">Quiz-App</h1>
       </header>
-      <main>
-      <form data-js="form">
+    
+      <main data-js="main">
+      
```

Comment:
```suggestion
      <form data-js="form">
```

Don't forget to close the form